drivers/hmis/app/models/hmis/hud/data_integrity/service_information_date_reconciler.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# Set service information date (new in 2026). It's not a nullable field so populate it with fa_start_date
+#
+class Hmis::Hud::DataIntegrity::ServiceInformationDateReconciler < Hmis::Hud::DataIntegrity::BaseReconciler
+  # @param [Hmis::Hud::Service] record
+  def call(record)
+    messages = []
+
+    # SSVF FA uses form input. Other services use start date s information date
+    record.information_date = record.fa_start_date if record.record_type != ssvf_financial_assistance
+    messages << 'information_date should be present' if record.information_date.nil?
```

Comment:
pending confirmation from HUD but I believe they walked back the InformationDate change (I just now noticed this), if so:
- Remove [this logic](https://github.com/greenriver/hmis-warehouse/blob/fde1cf6974779341f3ddc16aa12e9bc0e09248c7/drivers/hmis/app/models/hmis/hud/processors/service_processor.rb#L17-L23) which sets DateProvided=FAStartDate
- Remove new `informationDate` field from service.json form
- Remove `enable_when` condition from date provided on service.json form, so it is always collected even for V3
- (optional) add conditional instructional text for V3, explaining that the Date Provided should reflect the date that the financial assistance was identified as a need
- remove this reconciler